sdk/search/azure-search/src/main/java/com/azure/search/models/ImageDetail.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for ImageDetail.
+ */
+public enum ImageDetail {
```

Comment:
Same as with EntityCategory, I feel like these should be string constants instead of an enum

Reply:
Are there plans to update the current swagger spec in the coming days? As the model classes are generated code

Reply:
@navalev I'm not sure when it will happen (hopefully soon), but in any case I'm not inclined to block this PR on that.